Slo blog (#1137)
google/osv.dev: gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md

---
title: "Announcing OSV's Service Level Objectives"

> date: 2023-03-20T22:00:00Z

Author: ```suggestion
date: 2023-03-27T22:00:00Z
```

draft: false
author: OSV Team
---

We are excited to announce that OSV has published our new service level objectives (SLOs). 
<!--more-->

Even though OSV.dev is a free resource, we know that our users rely on us for accurate, up-to-date vulnerability information and we want to provide that on a consistent basis. We will be tracking the following:  

1. Website and API availability, measured across a seven day rolling window.   
2. Website and API latency.   
3. Data freshness.  

> Please refer to our [FAQ](https://osv.dev/about) for specific details.  

Reviewer: Is it possible to deep link to the relevant FAQ entry?

Reviewer: @oliverchang I think this is something to consider, are GitHub Discussions more fit-for-purpose as a dynamic FAQ than the FAQ as it currently exists on osv.dev?

Author: @andrewpollock I am in the process of reworking the docs, happy to meet to discuss. I think that the FAQ could find a new home in the docs and we could link to headings there. Open to other options too.

> Our performance will be shared via a quarterly blog post, with the hopes for a reporting dashboard in the future.  

Reviewer: ```suggestion
Performance against the SLO will be shared via a quarterly blog post, with the intention of a reporting dashboard in the future.  
```

Reviewer: Couple of minor tweaks
